### PR TITLE
minimos: r14 USB LED indication enabled.

### PR DESCRIPTION
**Pull request by sagarjminim**
SW-3102

### CODE REVIEW

**Review by bmastbergen — APPROVED** [review]

**Comment by sagarjminim** [comment]
Closing this ticket because according to the discussion, we are changing the mechanism of LED blinking.

**Comment by sagarjminim** [comment]
Reopening this PR as per the discussion.

**Comment by sagarjminim** [comment]
@bmastbergen @Tom-Keddie @richp-minim Please re-review the PR.

**Comment by sagarjminim** [comment]
@bmastbergen I have fixed the indentation. Please review.

File: samba-manager/files/samba-manager.init [hunk 1]
```diff
@@ -96,8 +96,7 @@ parse_config() {
 	config_get password ${cfg} password
 	config_get sharename ${cfg} sharename
 
-	mount=$(get_mount)
-	if [ $enabled -eq 1 -a ! -z $mount ] ; then
+	if [ $enabled -eq 1 ] ; then
 		add_samba_user $username $password
 		generate_samba_config $username $sharename $mount
```
Comment (bmastbergen) [inline samba-manager/files/samba-manager.init:101]: Looking at this again, how does mount get set to anything in this function?  I'm guessing this code doesn't actually work the way it is.  We may still need to call get_mount in this function to actually get the mount name to pass to generate_samba_config

**Review by bmastbergen — APPROVED** [review]

**Pull request merged**